microshift_running_apps/microshift-operators-olm.adoc

```diff
@@ -0,0 +1,62 @@
+:_mod-docs-content-type: ASSEMBLY
```

Comment:
@ShaunaDiaz one general comment: the reference you link is 4.14. you will change it to 4.15 in the future?

Reply:
Oh yes, good call, I will make those all attributes so the version rolls forward automatically.